package.json

```diff
@@ -1,37 +1,39 @@
   "main": "",
-  "repository": "https://github.com/witnet/witnet-pricefeed-examples",
+  "repository": "https://github.com/witnet/witnet-price-feed-examples",
   "author": "Witnet Devs <devs@witnet.io>",
   "license": "MIT",
   "private": false,
   "scripts": {
+    "avail": "npm run avail:examples && npm run avail:networks",
+    "avail:examples": "npm run compile:requests && node ./scripts/avail-examples.js 2>&1",
+    "avail:networks": "node ./scripts/avail-networks.js 2>&1",
     "console": "npx truffle console",
-    "clean": "npm run clean:flattened && node ./scripts/clean.js && npm run postinstall",
-    "clean:flattened": "node ./scripts/flattened-clean.js",
-    "compile": "npm run compile:contracts",
-    "compile:requests": "npx rad2sol",
-    "compile:contracts": "npm run clean:flattened && npm run postinstall && npm run compile:requests && npx truffle compile --all",
+    "clean": "npm run clean:build && npm rum clean:flattened && npm run postinstall",
+    "clean:build": "node ./scripts/clean.js build/",
+    "clean:flattened": "node ./scripts/clean.js flattened/",
+    "compile": "npm run clean && npm run compile:contracts",
+    "compile:requests": "npx rad2sol --disable-requests-contracts --disable-witnet-migrations --disable-user-contracts-migrations",
+    "compile:contracts": "npx truffle compile --all --config truffle-config.flattened.js",
     "coverage": "solidity-coverage",
-    "flatten": "node ./scripts/flatten.js 2>&1",
+    "flatten": "npm run compile && node ./scripts/flatten.js contracts/ 2>&1",    
     "fmt": "solhint --max-warnings 0 \"contracts/**/*.sol\" && eslint ./test && eslint ./scripts/*.js && solhint \"test/**/*.sol\"",
     "fmt!": "solhint --max-warnings 0 \"contracts/**/*.sol\" --fix && eslint ./test --fix && eslint ./scripts/*.js --fix && solhint \"test/**/*.sol\" --fix",
-    "migrate": "npm run clean:flattened && npx truffle migrate --reset",
-    "migrate:flattened": "node ./scripts/flattened-migrate.js 2>&1",
+    "migrate": "npm run clean:build && npm run compile:requests && node ./scripts/flattened-migrate.js 2>&1",
     "postinstall": "node ./scripts/postinstall.js 2>&1",
-    "solium": "solium -d contracts",
-    "solium:fix": "solium -d contracts --fix",
-    "test": "npm run compile:requests && npx truffle test",
-    "verify:flattened": "node ./scripts/flattened-verify.js 2>&1"
+    "test": "npx truffle test --config truffle-config.test.js"
   },
   "dependencies": {
+    "@eth-optimism/solc": "^0.7.6-alpha.1",
     "@openzeppelin/contracts": "^4.1.0",
     "adomedianizer": "https://github.com/adoracles/ado-contracts",
+    "lodash": "^4.17.21",
     "openzeppelin-solidity": "^2.4.0",
-    "witnet-ethereum-bridge": "https://github.com/witnet/witnet-ethereum-bridge",
-    "witnet-requests": "https://github.com/witnet/witnet-requests-js"
+    "witnet-ethereum-bridge": "git+https://github.com/guidiaz/witnet-ethereum-bridge#0.4.x-multi-interface",
```

Comment:
```suggestion
    "witnet-ethereum-bridge": "git+https://github.com/witnet/witnet-ethereum-bridge",
```

Reply:
As latest merged version was already tagged, this line will be changed to:
`"witnet-ethereum-bridge": "^0.4.1",`